fe/fe-core/src/main/java/com/starrocks/server/StorageVolumeMgr.java

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.server;
+
+import com.google.common.base.Preconditions;
+import com.staros.util.LockCloseable;
+import com.starrocks.common.AlreadyExistsException;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.storagevolume.StorageVolume;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.locks.ReadWriteLock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+public class StorageVolumeMgr {
+    private Map<String, StorageVolume> nameToSV = new HashMap<>();
+
+    private final ReadWriteLock rwLock = new ReentrantReadWriteLock();
+
+    private String defaultSV;
+
+    public void createStorageVolume(String name, String svType, List<String> locations, Map<String, String> params,
+                                    Boolean enabled, String comment) throws AlreadyExistsException, AnalysisException {
+        try (LockCloseable lock = new LockCloseable(rwLock.writeLock())) {
+            if (nameToSV.containsKey(name)) {
+                throw new AlreadyExistsException(String.format("Storage Volume '%s' already exists", name));
+            }
+            StorageVolume sv = new StorageVolume(name, svType, locations, params, enabled, comment);
+            nameToSV.put(name, sv);
+        }
+    }
+
+    public void removeStorageVolume(String name) {
+        try (LockCloseable lock = new LockCloseable(rwLock.writeLock())) {
+            Preconditions.checkState(nameToSV.containsKey(name),
+                    "Storage Volume '%s' does not exist", name);
+            Preconditions.checkState(!name.equals(defaultSV), "default storage volume can not be removed");
+            nameToSV.remove(name);
+        }
+    }
+
+    public void updateStorageVolume(String name, Map<String, String> params, Boolean enabled, String comment,
```

Comment:
Boolean -> boolean

Reply:
> Boolean -> boolean

`enabled` may be not set. So `enabled` can be null. But boolean can not represents this scenario